src/directory/generateDirectory.mjs

```diff
@@ -23,7 +23,8 @@ async function getMetaStringObj(filePath) {
     try {
       // Using JSON5 because the meta object is a "relaxed" JSON
       // JSON5 can parse the meta object without needing quotes around the object keys
-      const result = JSON5.parse(match[1]);
+      let metaObj = match[1].replaceAll('`',"'");
```

Comment:
replacing template strings to help parsing the meta object but this still breaks with nested ticks